packages/grafana-data/src/dataframe/index.ts

```diff
@@ -8,3 +8,9 @@ export * from './ArrayDataFrame';
 export * from './DataFrameJSON';
 export * from './frameComparisons';
 export { anySeriesWithTimeField, isTimeSeriesFrame, isTimeSeriesFrames } from './utils';
+export {
+  StreamingDataFrame,
+  closestIdx,
+  getStreamingFrameOptions,
+  getLastStreamingDataFramePacket,
```

Comment:
```suggestion
```
I don't think we need to expose these publicly yet.  We can internally import them with the full path